common/religions/void_group.txt

```diff
@@ -16,7 +16,7 @@ void_group = {
 	}
 	old_gods_worship = {
 		has_coa_on_barony_only = no
-		graphical_culture = norsegfx
+		graphical_culture = oldgodsgfx
```

Comment:
Map shields changed from Norse shields to Christian ones. You should assign Norse shields to this new graphical culture.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/100806535-0faaad00-344a-11eb-8110-bb9a7418fb31.png)

</details>

Reply:
how?